packages/wallets/src/features/cfd/modals/MT5ResetPasswordModal/MT5ResetPasswordModal.tsx

```diff
@@ -0,0 +1,135 @@
+                                                return (
+                                                    <WalletTextField
+                                                        autoComplete='current-password'
+                                                        errorMessage={
+                                                            hasCurrentPasswordFieldTouched && form.errors[field.name]
+                                                        }
+                                                        isInvalid={
+                                                            hasCurrentPasswordFieldTouched &&
+                                                            Boolean(form.errors[field.name])
+                                                        }
+                                                        label='Current password'
+                                                        name={field.name}
+                                                        onBlur={e => {
+                                                            setHasCurrentPasswordFieldTouched(true);
+                                                            field.onBlur(e);
+                                                        }}
+                                                        onChange={field.onChange}
+                                                        renderRightIcon={() => (
+                                                            <PasswordViewerIcon
+                                                                setViewPassword={setIsCurrentPasswordVisible}
+                                                                viewPassword={isCurrentPasswordVisible}
+                                                            />
+                                                        )}
+                                                        type={isCurrentPasswordVisible ? 'text' : 'password'}
+                                                        value={values.currentPassword}
+                                                    />
+                                                );
+                                            }}
+                                        </Field>
+                                        <WalletPasswordFieldLazy
+                                            label='New Password'
+                                            mt5Policy
+                                            name='newPassword'
+                                            onChange={handleChange}
+                                            password={values.newPassword}
+                                        />
+                                    </div>
+                                    <ul className='wallets-mt5-reset__requirements'>
+                                        {passwordRequirements.map((requirement, index) => (
+                                            <li key={index}>
```

Comment:
Lets not use index as keys 
https://blog.anja-stricker.de/why-using-index-as-key-in-reactjs-map-function-can-cause-problems-and-how-to-fix-them

Reply:
Who is Anja can we trust her 🔫 🤨 
jk will make the changes in the next commit!

Reply:
![image](https://github.com/binary-com/deriv-app/assets/104053934/372f9940-dedb-4059-930b-b3d2288d3f01)

Reply:
![image](https://github.com/binary-com/deriv-app/assets/103104395/1b0fd761-f49a-4457-9bce-3dc88f90da32)